src/main/kotlin/com/launchdarkly/intellij/toolwindow/FlagNodeViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.launchdarkly.intellij.toolwindow
+
+import com.launchdarkly.api.model.FeatureFlag
+import com.launchdarkly.api.model.FeatureFlags
+import com.launchdarkly.intellij.LDIcons
+import com.launchdarkly.intellij.featurestore.FlagConfiguration
+
+class FlagNodeViewModel(
+    val flag: FeatureFlag,
+    val flags: FeatureFlags,
+    val flagConfig: FlagConfiguration?,
+) {
+    val description = flag.description
+    val isEnabled = flagConfig?.on == true
+    val isDisconnected = flagConfig == null
+    val targets = flagConfig?.targets ?: listOf()
+    val prereqFlags = flagConfig?.prerequisites ?: listOf()
```

Comment:
Thank you for this awesome ViewModel! It's a great design. For prereqs, perhaps it's also useful to have a property called `hasPrereqs` so consumers of this vm can all avoid doing `viewModel.prereqFlags.isNotEmpty()`?

Reply:
Maybe the above also applies for `targets` => `hasTargets` and `numRules` => `hasRules`?

Reply:
updated, thanks! Did the same for tags